src/parsing.jl

```diff
@@ -369,11 +369,11 @@ _DEFAULT_TYPE_MAP[:tstzrange] = Interval{ZonedDateTime}
 _DEFAULT_TYPE_MAP[:daterange] = Interval{Date}
 
 # Matches anything but the start or end of an interval or a comma
-const RANGE_ITEM = "[^\\[\\(\\]\\),]+"
-# Makes sure the string starts and ends with a bracket or parentheses, has two items in the
-# interval, and a comma to separate the two items.
+const RANGE_ITEM = "[^\\[\\(\\]\\),]*"
+# Makes sure the string starts and ends with a bracket or parentheses and a comma separates
+# the items in the interval
 const RANGE_REGEX = Regex("^([\\[\\(])($RANGE_ITEM),($RANGE_ITEM)([\\]\\)])\$")
-get_inclusivity(ch) = ch in ("[", "]") ? true : false
+get_bounds_type(ch) = ch in ("[", "]") ? Closed : Open
```

Comment:
Very minor but each time I read this I think `ch` is a `Char` which isn't the case:
```julia
julia> '[' in ("[", "]")
false
```
```suggestion
get_bounds_type(ch::AbstractString) = ch in ("[", "]") ? Closed : Open
```